avocado/utils/disk.py

```diff
@@ -154,3 +154,22 @@ def get_filesystem_type(mount_point='/'):
             _, fs_file, fs_vfstype, _, _, _ = mount_line.split()
             if fs_file == mount_point:
                 return fs_vfstype
+
+
+def is_root_device(device):
+    """
+    check for root disk
+
+    :param device: device to check
+    :returns: True or False, True if given device is root disk
+              otherwise will return False.
+    """
+    cmd = "lsblk --j -o MOUNTPOINT,PKNAME"
+    output = process.run(cmd)
+    result = json.loads(output.stdout_text)
+    for item in result['blockdevices']:
+        if item['mountpoint'] == "/":
+            res = str(item['pkname'])
```

Comment:
Do you expect more than one mount point with on "/"? I don't see a use case for that, do you have one in mind?

If not, what do you think about:

```
diff --git a/avocado/utils/disk.py b/avocado/utils/disk.py
index 053d3bc1..6c3a737b 100644
--- a/avocado/utils/disk.py
+++ b/avocado/utils/disk.py
@@ -167,8 +167,6 @@ def is_root_device(device):
     output = process.run(cmd)
     result = json.loads(output.stdout_text)
     for item in result['blockdevices']:
-        if item['mountpoint'] == "/":
-            res = str(item['pkname'])
-    if res in device:
-        return True
+        if item['mountpoint'] == "/" and device == str(item['pkname']):
+            return True
     return False
```

Reply:
this is fine for me.